pkg/localmetrics/localmetrics.go

```diff
@@ -15,22 +15,97 @@
 package localmetrics
 
 import (
+	"net/http"
+	"time"
+
 	"github.com/prometheus/client_golang/prometheus"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("localmetrics")
+
+const (
+	apiEndpoint = "https://api.pagerduty.com/users"
 )
 
 var (
-	metricPlaceholder = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "placeholder",
-		Help: "Placeholder",
+	MetricPagerDutyCreateFailure = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "metricPagerDutyCreateFailure",
```

Comment:
Very small nit, but most of the names are usually in the form of `metric_pagerduty_create_failure` and the Help would be something like `Reports the status of PD service creation`